e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -271,27 +309,38 @@ describe("viem send tx", () => {
 		}
 	}).timeout(10_000);
 
-	it("send fee currency tx with just high enough gas price", async () => {
+	it("send fee currency tx with just high enough gas price mine", async () => {
+		const rate = await getRate(process.env.FEE_CURRENCY);
 		const block = await publicClient.getBlock({});
-		// Actually, the base fee will be a bit lower next block, since our blocks
-		// are always mostly empty. But the difference will be much less than the
-		// exchange rate of 2, so the test will still check that we take the fee
-		// currency into account everywhere.
-		const maxFeePerGas = block.baseFeePerGas / 2n;
+		const maxFeePerGas = rate.toFeeCurrency(block.baseFeePerGas)+2n;
 		const request = await walletClient.prepareTransactionRequest({
 			account,
 			to: "0x00000000000000000000000000000000DeaDBeef",
 			value: 2,
-			gas: 90000,
-			feeCurrency: process.env.FEE_CURRENCY2,
-			maxFeePerGas,
-			maxPriorityFeePerGas: 1n,
+			gas: 171000,
+			feeCurrency: process.env.FEE_CURRENCY,
+			maxFeePerGas: maxFeePerGas,
+			maxPriorityFeePerGas: 2n,
 		});
 		const signature = await walletClient.signTransaction(request);
 		const hash = await walletClient.sendRawTransaction({
 			serializedTransaction: signature,
 		});
 		const receipt = await publicClient.waitForTransactionReceipt({ hash });
 		assert.equal(receipt.status, "success", "receipt status 'failure'");
-	}).timeout(5_000);
+	}).timeout(10_000);
```

Comment:
Is this necessary? With 1 second block times, 5 seconds should be plenty.

I can understand the increase for testing against a live network, but when iterating on local changes, quick failure is a nice feature. Most failures during development are not of the "tx does not get included" kind though.

Reply:
So this is the running time on alfajores.

```
    ✔ send fee currency tx with just high enough gas price (4998ms)
```

I generally like to have a reasonable sized buffer for timeouts to avoid failures based on minor resource availability fluctuations. So I'd argue it is necessary.

 If you feel strongly about this we could also add custom timeouts per network.

Reply:
In that case, increasing the timeout makes sense. I expected the test to be much faster on alfajores.